src/main/java/org/folio/innreach/domain/service/impl/InnReachTransactionActionServiceImpl.java

```diff
@@ -283,6 +283,9 @@ private void updateTransactionOnLoanClaimedReturned(StorageLoanDTO loan, InnReac
     log.info("Updating patron transaction {} on the claimed returned loan {}", transaction.getId(), loan.getId());
 
     transaction.setState(CLAIMS_RETURNED);
+    var itemHold = transaction.getHold();
+    itemHold.setPatronId(null);
+    itemHold.setPatronName(null);
```

Comment:
This logic should be applied for owning site transaction (Patron Hold is on borrowing site) when the "claimsreturned" message is received from the central server.